resources/definitions/kosher_duplication.def.json

```diff
@@ -0,0 +1,51 @@
+{
+    "version": 2,
+    "name": "Kosher_Duplication",
```

Comment:
Is there an underscore in the printer name? Seems a bit weird.
I can't find any information on this printer online to verify it against. I'd guess it's just a duplication mode that the printer can do with 2 print heads. Then maybe just a space is what you intended.